core/citrus-base/src/main/java/org/citrusframework/validation/matcher/DefaultValidationMatcherLibrary.java

```diff
@@ -16,28 +16,7 @@
 
 package org.citrusframework.validation.matcher;
 
-import org.citrusframework.validation.matcher.core.ContainsIgnoreCaseValidationMatcher;
-import org.citrusframework.validation.matcher.core.ContainsValidationMatcher;
-import org.citrusframework.validation.matcher.core.CreateVariableValidationMatcher;
-import org.citrusframework.validation.matcher.core.DatePatternValidationMatcher;
-import org.citrusframework.validation.matcher.core.DateRangeValidationMatcher;
-import org.citrusframework.validation.matcher.core.EmptyValidationMatcher;
-import org.citrusframework.validation.matcher.core.EndsWithValidationMatcher;
-import org.citrusframework.validation.matcher.core.EqualsIgnoreCaseValidationMatcher;
-import org.citrusframework.validation.matcher.core.GreaterThanValidationMatcher;
-import org.citrusframework.validation.matcher.core.IgnoreNewLineValidationMatcher;
-import org.citrusframework.validation.matcher.core.IgnoreValidationMatcher;
-import org.citrusframework.validation.matcher.core.IsNumberValidationMatcher;
-import org.citrusframework.validation.matcher.core.LowerThanValidationMatcher;
-import org.citrusframework.validation.matcher.core.MatchesValidationMatcher;
-import org.citrusframework.validation.matcher.core.NotEmptyValidationMatcher;
-import org.citrusframework.validation.matcher.core.NotNullValidationMatcher;
-import org.citrusframework.validation.matcher.core.NullValidationMatcher;
-import org.citrusframework.validation.matcher.core.StartsWithValidationMatcher;
-import org.citrusframework.validation.matcher.core.StringLengthValidationMatcher;
-import org.citrusframework.validation.matcher.core.TrimAllWhitespacesValidationMatcher;
-import org.citrusframework.validation.matcher.core.TrimValidationMatcher;
-import org.citrusframework.validation.matcher.core.WeekdayValidationMatcher;
+import org.citrusframework.validation.matcher.core.*;
```

Comment:
please try reverting this. in IntelliJ, you can do this by settings:

![image](https://github.com/user-attachments/assets/cb4e42af-d7b0-4fa9-96fc-fdfdd576fc88)